assets/src/scss/base/_typography.scss

```diff
@@ -102,7 +102,7 @@ a --link-- {
   text-decoration: none;
 
   &:focus {
-    border: 2px solid rgba(0, 106, 255, 0.4);
+    outline: 2px solid rgba(0, 106, 255, 0.4);
```

Comment:
`:focus-visible` has somewhat nicer behavior in our case, because it avoids the hover style being added as you mouse-click or tap the button to navigate. In that case the outline flashes for a moment and looks weird.

However it doesn't have full browser support yet, specifically Safari. https://caniuse.com/css-focus-visible

So maybe something we can get back to later.

Reply:
Oh btw, while the normal links are not in scope of this ticket, I needed to fix this because I had to remove `a:focus-visible { outline: 0 }` below to get buttons working correctly.

Reply:
Does it make sense to use a `@supports` clause so at least we have the nice behavior on all other browsers?

Reply:
I found another fix in the meanwhile that plays well with non-supporting browsers, using `a:focus:not(:focus-visible)` to remove the outline again.

If the browser doesn't support `:focus-visible`, unlike what you might expect the `:not()` selector to do, the selector won't match. So those would still keep the focus styles.

Still looking for a confirmation of this browser behavior in the specs but it seems to work.

Reply:
It's similar to what's suggested in the example here: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#selectively_showing_the_focus_indicator